samples/RXPTest/src/Tests/ViewBasicTest.tsx

```diff
@@ -106,6 +110,59 @@ class BasicView extends RX.Component<RX.CommonProps, RX.Stateless> {
                           'I have differing border radii' }
                     </RX.Text>
                 </RX.View>
+
+                <RX.View 
+                    style={ _styles.explainTextContainer } 
+                    key={ 'explanation2' }
+                >
+                    <RX.Text style={ _styles.explainText }>
+                        { 'Should be read by screen reader as \"Alpha Beta Gamma Delta\" ' + 
+                          'and screen reader should not be able to get to individual texts.' }
+                    </RX.Text>
+                </RX.View>
+                <RX.View style={ _styles.labelContainer }>
+                    <RX.View 
+                        style={ _styles.labelContainer } 
+                        importantForAccessibility={ RX.Types.ImportantForAccessibility.Yes }
+                    >
+                        <RX.Text style={ _styles.labelText }>
+                            { 'Alpha' }
+                        </RX.Text>
+                        <RX.View style={ _styles.labelContainer }>
+                            <RX.Text style={ _styles.labelText }>
+                                { 'Beta' }
+                            </RX.Text>
+                            <RX.View 
+                                style={ _styles.labelContainer } 
+                                accessibilityLabel={ 'Gamma' }
+                            >
+                                <RX.Text style={ _styles.labelText }>
+                                    { 'Orange' }
+                                </RX.Text>
+                            </RX.View>
+                        </RX.View>
+                        <RX.Text style={ _styles.labelText }>
+                            { 'Delta' }
+                        </RX.Text>
+                        <RX.Text 
+                            style={ _styles.labelText } 
+                            importantForAccessibility={ RX.Types.ImportantForAccessibility.No }
+                        >
+                            { 'Epsilon' }
```

Comment:
Should we verify the behavior of nesting something inside of a `Text` with `importantForAccessibility=No`?

Reply:
In the current implementation if you try two-level nesting inside RX.Text, you get an exception

Reply:
This does not seem to be nesting anything inside RX.Text. Can you elaborate?

---
In reply to: [161927956](https://github.com/Microsoft/reactxp/pull/431#discussion_r161927956) [](ancestors = 161927956)

Reply:
That would be Text tests, probably should go to TextTest.tsx - not part of this View test PR.

---
In reply to: [161925535](https://github.com/Microsoft/reactxp/pull/431#discussion_r161925535) [](ancestors = 161925535)

Reply:
There isn't. I am just replying to the proposal to test nesting inside Text. Two level nesting will cause exception in the present implementation, i.e.
<RX.Text>
   xxxx
   <RX.View>
    <RX.Text>
     yyyyyy

Reply:
Sorry, CF does not show your replies as part of thread. Can you use CF to review the PRs?

---
In reply to: [161929225](https://github.com/Microsoft/reactxp/pull/431#discussion_r161929225) [](ancestors = 161929225)